packages/3d-web-client-core/src/rendering/composer.ts

```diff
@@ -539,6 +539,8 @@ export class Composer {
       envValues.ambientLight.ambientLightIntensity =
         this.environmentConfiguration.ambientLight.intensity;
       this.setAmbientLight();
+    } else {
```

Comment:
This function could just be called outside the if

Reply:
@MarcusLongmuir fixed